web/docs/guides/auth/auth-facebook.mdx

```diff
@@ -0,0 +1,116 @@
+---
+id: auth-facebook
+title: "Facebook Login"
+description: Setting up Facebook Login
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Setting up Facebook Login
+
+## Description
+
+Adding a Facebook Login button to your application is easy if you follow these steps.
```

Comment:
Remove these lines - we don't need to tell them that it's easy, we just need to make it easy